cinema/views.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from  rest_framework import status
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
```

Comment:
Using `else` here is not explicit about handling the POST request. Consider using `elif request.method == 'POST':` for clarity and to avoid potential issues if more methods are added in the future.